openapi-cli/src/test/java/io/ballerina/openapi/generators/client/UtilGenerationTests.java

```diff
@@ -188,5 +189,24 @@ private boolean checkUtil(List<String> invalidFunctionNames, SyntaxTree utilSynt
         }
         return false;
     }
+
+    private boolean checkDefaultUtil(String validFunctionName, SyntaxTree utilSyntaxTree) {
```

Comment:
Since this util function is only used for one test case, cant we compare generated and expected syntax tree without traversing through the syntax tree ?